Makefile

```diff
@@ -196,9 +197,13 @@ verify:
 	@echo "verify: Started"
 	@echo "verify: Completed"
 
-test:
-	@echo "test: Started"
-	@echo "test: Completed"
+SKIP_TESTS ?=
+test: build-dirs
+ifneq ($(SKIP_TESTS), 1)
+	@$(MAKE) shell CMD="-c '\
+	     VDDK_LIBS=$(VDDK_LIBS_CONTAINER) \
```

Comment:
You don't need a configurable VDDK_LIDS argument here, since it is usually placed in a pretty fixed location. To check whether vddk libs is ready before running the `go test` command, you can do it in hack/test.sh as below
```
ifeq (,$(wildcard $(VDDK_LIBS)))
	$(error "$(VDDK_LIBS) cannot find vddk libs in path, please reference to: https://github.com/vmware-tanzu/astrolabe/tree/master/vendor/github.com/vmware/gvddk#dependency")
endif
```
Meanwhile, you can use the relative path.

What you actually need is to provide a configurable test target here, so that devs may want to test a single test case while debugging.